en/docs/_ci/semaphore.md

```diff
@@ -0,0 +1,10 @@
+[Semaphore](https://semaphoreci.com/) has Yarn pre-installed for all
+supported Node.js versions, and no user interaction is required for the Yarn
+cache to work.
+
+To assure that your local Yarn version matches the one on Semaphore, add the
+line below to your setup commands, in Project Settings.
+
+```sh
+npm install -g yarn@<version>
```

Comment:
Please avoid installing Yarn via npm in the documentation. There's a number of issues with installing Yarn via npm so it should be avoided.  Could you please use the installation script instead?